dev-lang/rust/rust-1.74.1.ebuild

```diff
@@ -426,7 +443,9 @@ src_configure() {
 		debuginfo-level-tests = 0
 		backtrace = true
 		incremental = false
-		default-linker = "$(tc-getCC)"
+		$(if ! tc-is-cross-compiler; then
+			echo "default-linker = \"$(tc-getCC)\""
+		fi)
```

Comment:
It'll be nice to have explanation why it's not like that, so it won't get reverted, similar and similar for `CMAKE` llvm stuff.